package.json

```diff
@@ -31,6 +17,29 @@
     "node": "^10",
     "yarn": "^1.16.0"
   },
+  "scripts": {
+    "build": "tsc --build",
+    "dev": "concurrently --kill-others \"yarn watch\" \"yarn serve\"",
+    "deploy": "yarn install && yarn demolish && yarn build",
+    "demolish": "rimraf dist",
+    "serve": "nodemon --watch dist --watch config --delay 4 dist/_main.js",
+    "start": "node dist/_main.js",
+    "lint": "yarn lint-tslint && yarn lint-prettier",
```

Comment:
This runs `lint-prettier` twice since you added the `prettier-plugin` to `tslint`